distribution/docs/src/main/resources/content/_installing/high-availability-initial-setup-exceptions.adoc

```diff
@@ -0,0 +1,63 @@
+:title: High Availability Initial Setup Exceptions
+:type: subInstalling
+:status: published
+:parent: High Availability Initial Setup
+:project: ${branding}
+:order: 00
+
+==== {title}
+
+These steps are handled differently for the initial setup of a Highly Available Cluster.
+
+===== HTTP Proxy Integration
+
+In order to integrate with an HTTP proxy, the ${branding} node's system properties must be changed to publish the correct port to external systems and users.
+This must be done before installing the first ${branding} node.
+There are two internal port properties that must be changed to whatever ports the ${branding} will run on on its system.
+Then there are two external port properties that must be changed to whatever ports the HTTP proxy is forwarding traffic through.
+
+[WARNING]
+====
+Make sure that the HTTP proxy is already running and forwarding traffic on the chosen ports before starting the ${branding}.
```

Comment:
HTTP proxy is the term we should use above instead of loadbalancer

Reply:
Should we use HTTP proxy or failover proxy?

Reply:
I personally prefer HTTP proxy, but am open if failover proxy is preferred.

Reply:
Either. As long as we're consistent.